packages/components/layout/src/Layout.styles.ts

```diff
@@ -20,28 +29,107 @@ export const getLayoutMaxWidthStyles = ({
     boxShadow: withBoxShadow
       ? '0px 6px 16px -2px rgba(25, 37, 50, 0.1), 0px 3px 6px -3px rgba(25, 37, 50, 0.15)'
       : 'unset',
-    borderRadius: `10px 10px 0 0`,
-    margin: `0 ${tokens.spacingM}`,
+    borderRadius: '10px 10px 0 0',
+    margin: `0 auto`,
+    width: `calc(100% - ${tokens.spacingM} * 2)`,
```

Comment:
Could you please comment on the rationale behind this calculation?

Reply:
I removed the outer wrapper that was there to center the main container via flexbox, because it was messing with the container heights. Intead I am using now our good old friend `margin: 0 auto` but we want that even on  smaller screens, there is a 1rem gap between the white container and the rest of it, i can not set the width to 100%. It needs to be `100% - applied margin`

happy to add a comment to the code.

Reply:
Thank you for explaining, I think it is worth storing `${tokens.spacingM} * 2` in a variable to make this clear + add a comment, yes 💯